src/compounds/Messages/MessageCardBody.jsx

```diff
@@ -52,7 +52,11 @@ MessageCardBody.propTypes = {
       }),
     ),
   }).isRequired,
-  isLatestMessage: PropTypes.bool.isRequired,
+  isLatestMessage: PropTypes.bool,
```

Comment:
is this another change because of something loading?

Reply:
it was actually because this prop was just being used for a style and it wasn't really necessary, i just refactored and removed it altogether though because it was just removing padding for when it was the latest message. i checked it and it looks better with the padding anyways though